src/test/resources/application.yml

```diff
@@ -25,10 +25,15 @@ fabric:
     rest:
       apikey: ePVYHwAaQ0V1XOTX6U
 kafka:
-  integration:
-    brokerHost: localhost:9093
-    groupId: fabric-consumer
-    topic: hlf-integration-topic
+  integrationList:
```

Comment:
[nitpick] `integrationList` is not giving the right explanation to what this section of configuration is supposed to do. Can we still keep the name to `integration` or make sound like `integration-topics`, `integration-points` etc?